package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@bitcoin-design/bitcoin-icons-react": "^0.1.9",
+    "@headlessui/react": "^1.7.15",
```

Comment:
👍
Was playing around with it once as well

Reply:
Alby is using this and it works pretty nice